scripts/test

```diff
@@ -51,11 +51,16 @@ compose() {
 (
   cd "${projectRoot}"
 
+  # create test output folder
+  mkdir -p test-output
```

Comment:
If this folder doesn't already exist before the tests are first run then it will be created by the test container.  But because it's created by a container, it will be created with root permissions and the tests won't actually have permission to write to them.  

So wise to ensure it's created already to avoid that pain.